hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/meta/MetaManager.java

```diff
@@ -242,7 +251,19 @@ public <T> void listenGraphClear(Consumer<T> consumer) {
     }
 
     public <T> void listenSchemaCacheClear(Consumer<T> consumer) {
-        this.graphMetaManager.listenSchemaCacheClear(consumer);
+        if (isListenerInitialized(SCHEMA_CLEAR_KEY)) {
```

Comment:
not sure why registering listener when isListenerInitialized(), do you mean isListenerNotInitialized()?

Reply:
yes，it is isListenerNotInitialized()